lib/smart_proxy_host_reports/ansible_processor.rb

```diff
@@ -82,46 +74,56 @@ def spool_report
 
   private
 
-  def log_level(log)
-    if log["failed"]
-      "err"
-    elsif log["changed"]
-      "notice"
-    else
-      "info"
+  def process_keywords(result)
+    if result["failed"]
+      add_keywords("HasFailure", "AnsibleTaskFailed:#{result["task"]["action"]}")
+    elsif result["changed"]
+      add_keywords("HasChange")
     end
   end
 
-  def log_source(log)
-    # Temporary until  is resolved
-    log["module"]
+  def process_level(result)
+    if result["failed"]
+      result["level"] = "err"
+    elsif result["changed"]
```

Comment:
```suggestion
    elsif result["result"]["changed"]
```
The `changed` key is not on the same level as `failed` key. Have no idea why, but...

![ScreenShot-1633553870196](https://user-images.githubusercontent.com/32508194/136282091-85ccef21-0ebb-426f-81bd-44ef85d473a5.png)

Reply:
Yeah you are right, now it works as expected.

Reply:
JFYI "result" and "task" are coming from Ansible unchanged, some keys in the root are created in the callback.py. And some others will be added by this plugin.

Reply:
You probably forgot to update this line in the same way as here: https://github.com/theforeman/smart_proxy_host_reports/pull/5/files#diff-73551ae351ce5f8b8df387a12af7654fa8789cfed325641f8cbf48311e49b028R80

Reply:
Yeeah right.